ddtrace/contrib/internal/langchain/patch.py

```diff
@@ -1203,6 +1363,9 @@ def patch():
         wrap("langchain", "embeddings.OpenAIEmbeddings.embed_documents", traced_embedding(langchain))
     else:
         from langchain.chains.base import Chain  # noqa:F401
+        from langchain_core import messages  # noqa: F401
+        from langchain_core import output_parsers  # noqa: F401
+        from langchain_core import prompt_values  # noqa: F401
```

Comment:
I assume these are due to import errors on langchain's end (not being able to directly access them without importing first?)

Reply:
tbh i just added them by default. let me try and remove them and see what happens

Reply:
looks like it works without them 🫡